sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-student-unmet-need-report.sql

```diff
@@ -0,0 +1,68 @@
+INSERT INTO
+  sims.report_configs (report_name, report_sql)
+VALUES
+  (
+    'Student_Unmet_Need_Report',
+    'SELECT
+      users.first_name AS "Student First Name",
+      users.last_name AS "Student Last Name",
+      sin_validations.sin AS "SIN",
+      applications.student_number AS "Student Number",
+      users.email AS "Student Email Address",
+      students.contact_info ->> ''phone'' AS "Student Phone Number",
+      institution_locations.institution_code AS "Institution Location Code",
+      institution_locations.name AS "Institution Location Name",
+      applications.application_number AS "Application Number",
+      student_assessments.assessment_date AS "Assessment Date",
```

Comment:
Does the business require the timestamp or date only?

Reply:
Outside the PR we need to check with the business the expectation for the date and timestamps, we probably should have those converted to PDT/PST but it may not be so important depending on the case, hence, up to biz to decide.